### PR TITLE
Jester of Nihil Part 2 (of 3)

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Partially completes the jester of nihil abnormality and boss fight. There is a lot going on here.

- Completes work mechanics for the jester of nihil.
- Adds an admin verb to spawn boss abnormalities. Currently includes apocalypse bird, yin and yang, and the jester of nihil.
- Adds basic code for the magical girls so that they function properly during the jester of nihil fight.
- Adds approximately 40%  of the code and mechanics expected for the boss fight, putting it in a rudimentary but functional state.

TODO in another PR :
- Special mechanics to prevent the boss from being damaged under certain conditions.
- Special attacks for every boss phase, some of the more complicated attacks that will take time to code.
- Additional attacks for the magical girls during the fight.
- Additional buffs for the fight.
- Enable nihil as an abnormality choice.
- "Cutscenes" for certain events during the fight.

## Why It's Good For The Game
Getting this merged quickly gives administrators access to a useful verb and a somewhat functional new feature.

## Changelog
:cl:
add: Added jester of nihil fight (incomplete)
add: Added spawn boss abnormality verb
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -861,6 +861,20 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
...
+	var/list/bosslist = list("Apocalypse Bird","Ying and Yang","The Jester of Nihil") //TBA : General bee, nobody is.
+	if(!SSticker.HasRoundStarted())
+		alert("Wait until the game starts")
+		return
+	var/M = input(usr,"Who would you like to spawn? (This will take a few seconds to load)","Select a boss") as null|anything in bosslist
+	if(!M)
+		return
+	SSlobotomy_events.SpawnEventAbnos(bosslist.Find(M))
+	SSblackbox.record_feedback("tally", "admin_verb", 1, "Spawned abnormality boss [M]")
+
```
Comment (LanceSmites328) [inline code/modules/admin/verbs/debug.dm:877]: Appreciated; Though I believe one of two things should be done here. Either the `bosslist` becomes associative with the defines or equivalent defines of the event defines in the Lobotomy Corp Events subsystem OR a comment is here noting that the index of the list is equivalent to the defined value in the Lobotomy Corp Events subsystem.

File: code/modules/paperwork/records/info/aleph.dm [hunk 2]
```diff
@@ -136,12 +136,13 @@
 	abno_type = /mob/living/simple_animal/hostile/abnormality/nihil
 	abno_code = "O-01-150"
 	abno_info = list(
-		"When the work result was Neutral, the Qliphoth Counter decreased at a normal probability.",
+		"When the work result was good, the Qliphoth counter increased.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/aleph.dm:139]:
```suggestion
		"When the work result was Good, the Qliphoth counter increased.",
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/debug.dm (hunk 1 above)
Comment (Coxswain-Navigator) [reply code/modules/admin/verbs/debug.dm:877]: There is already a comment, "//The order and contents of the bosslist should match the defines set in lobotomy_events". However, I agree that this is normally a valid concern.

**Comment by Kitsunemitsu** [comment]
Guh
Time for me to fucking read this PR finally

**Review by Kitsunemitsu — APPROVED** [review]
Approved. Finally got around to reading this. 

A bit of copypasta but that's honestly fine, as it's something that affects 4 files.
Expect that if we add statues to more abnos for some fuckign reason that we will have to atomize it.

**Comment by Kitsunemitsu** [comment]
Merge whenever, the stage is yours.

**Pull request merged**